city_temperature/main.py

```
@@ -0,0 +1,19 @@
+from fastapi import FastAPI
+from city_temperature.database import Base, engine
+from city_temperature.routers import city_router, temperature_router
+
+Base.metadata.create_all(bind=engine)

```

Calling `Base.metadata.create_all(bind=engine)` during module import will execute schema creation as a side-effect when the module is imported. Consider moving this to a FastAPI startup event (using `@app.on_event("startup")`) so database initialization is part of the application lifecycle and easier to control in tests and deployments. This also lets you handle startup errors more gracefully (log and fail fast).